log_bucket/main.tf

```diff
@@ -0,0 +1,43 @@
+resource "aws_s3_bucket" "logs" {
+  acl    = "log-delivery-write"
+  bucket = "${var.bucket}"
+}
+
+data "aws_iam_policy_document" "logs_bucket_policy" {
+
+  statement {
+    sid       = "Allow LB to write logs"
+    actions   = ["s3:PutObject"]
+    resources = ["arn:aws:s3:::${aws_s3_bucket.logs.bucket}/${var.alb_logs_path}*"]
+    principals {
+      type        = "AWS"
+      identifiers = ["arn:aws:iam::156460612806:root"]
```

Comment:
Are intending to hard code these accounts? I didn't see them in our organization so I'm not sure what they are?